### DISCUSSION TITLE
Prototype Digital Watch with ESP32 and U8G2

### DISCUSSION THREAD

**Discussion by Fishbone69** [olikraus/u8g2#2232] [Show and tell]
First, I want to say I was heavily inspired by this:  https://youtu.be/KrUJlZK1ct0

I wanted to see if I could make my own.  This is what I have so far:  https://youtu.be/n2ouRHl6IEY

No games on mine yet, but I am making headway.  I just soldered an esp32 dev board into a pcb that I designed for the display.  I just submitted plans for a new pcb that does not use a dev board but just an esp32 chip.  It should make mine much more compact.  I know mine isn't as polished as Zac's, but since I had never programmed a microcontroller prior to Feb of this year AND never designed a pcb before, I am happy with my progress.  What do you guys think?

Also want to thank Ollie and all of the work he has done on u8g2 and the patience he had answering my noob questions.

Fish

**Comment by olikraus** [comment]
Very cool. I like the time offset world map.
